server/src-exec/Migrate.hs

```diff
@@ -432,4 +432,4 @@ migrateCatalog migrationTime = migrateFrom =<< getCatalogVersion
                 UPDATE "hdb_catalog"."hdb_version"
                    SET "version" = $1,
                        "upgraded_on" = $2
-                    |] (curCatalogVer, migrationTime) False
```

Comment:
I’m not sure why a bunch of these files in the server code had their trailing newlines stripped, but could you revert these changes on your branch? (That way I won’t be a code owner on this PR, and you won’t have to wait for me to review it.)

Reply:
Done.